docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,472 @@
+    }
+}
+----
+<1> Path of the resource
+<2> HTTP method
+<3> Expected response content type
+<4> Returning text payload with status 200
+
+JAX-RS is very powerful tool where you can register your own message body writers, readers, filters or exception mappers.
+In both, Helidon and Oracle Weblogic Server, is {jersey-link} used as an JAX-RS implementation.
+
+For further reading on how to create and deploy JAX-RS RESTful resources in Helidon continue to xref:../../mp/jaxrs/jaxrs-applications.adoc[JAX-RS applications.]
+For developing and deploying JAX-RS resources on Oracle Weblogic Server follow {wls-jaxrs-link} documentation.
+
+=== JAX-RS Client [[REST-Client]]
+
+JAX-RS provides convenient client API for calling RESTful resources.
+Client allows you to prepare and execute RESTful request call with a simple builder pattern API.
+
+[source,java]
+.JAX-RS client example
+----
+Client client = ClientBuilder.newClient();
+String res = client
+    .target("http://localhost:8080") //<1>
+    .path("/greet") //<2>
+    .request("text/plain") // <3>
+    .get(String.class); // <4> <5>
+----
+<1> Creates new WebTarget with default root url
+<2> Prepare request to particular context path
+<3> Set expected response content type
+<4> Execute GET request and block until response is received
+<5> Parameter sets expected response payload type, available body readers are used for parsing to the correct type
+
+Again you can register your own message body writers, readers, filters or exception mappers.
+
+For further reading on how to create JAX-RS clients in Helidon continue to xref:../../mp/jaxrs/jaxrs-client.adoc[JAX-RS client.]
+For developing and deploying JAX-RS clients on Oracle Weblogic Server follow {wls-jaxrs-client-link} documentation.
+
+=== References [[REST-Services-References]]
+* xref:../../mp/introduction/microprofile.adoc[Helidon MP Tutorial]
+* xref:../../mp/guides/quickstart.adoc[Helidon MP Quickstart]
+* {wls-jaxrs-link}
+* {wls-jaxrs-client-link}
+* {jersey-link}
+
+== JMS Integration [[JMS-Integration]]
+
+- <<JMS-Integration-Overview, Overview>>
+- <<JMS-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<JMS-Integration-Usage, Usage>>
+- <<JMS-Integration-References, References>>
+
+=== Overview [[JMS-Integration-Overview]]
+Oracle Weblogic Server provides Java Message Service (JMS) and acts as a messaging broker accessible even from outside the cluster.
+To access Oracle Weblogic JMS Server from outside, client library is needed.
+Helidon JMS connector for reactive messaging can be configured to use {wls-thin-link} for either consuming or
+emitting messages.
+
+{wls-thin-link} can be obtained from multiple sources.
+Legacy versions of thin client can be found in server/lib directory(`WL_HOME/server/lib/wlthint3client.jar`) of any WebLogic Server installation.
+
+WARNING: Helidon supports {jakarta-version} and legacy versions of `javax` based thin T3 client wouldn't work correctly when on it's classpath.
```

Comment:
Please review English of this warning. I believe something like the following will be more clear:
Warning: Helidon supports {jakarta-version}. Previous versions of Java EE which use object with 'javax' class signatures will not work directly. Placing a 'javax' version of the T3 Thin client onto the Helidon-service class path will not work. For proper integration, the 'javax' version of T3 Thin client must be used, only with specialized xref:../../mp/reactivemessaging/weblogic.adoc[WebLogic JMS Connector].